lib/Doctrine/Common/Annotations/DocLexer.php

```diff
@@ -126,4 +128,12 @@ protected function getType(&$value)
 
         return $type;
     }
+
+    /** @return array{value: int|string, type:self::T_*|null, position:int} */
+    public function peek(): array
+    {
+        $token = parent::peek();
+
+        return (array) $token;
```

Comment:
Should the array type always be forced here? In `v1.13.x` that was not the case because the method was inherited from the parent class. The docblock does not state that an empty array could be returned, but that is exactly what will be returned when the parent returns `null` right now.

Reply:
Fixed in https://github.com/doctrine/annotations/pull/463 :tada: